quality/errorprone/android.gradle

```diff
@@ -1,10 +1,16 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.4.6"
-  testAnnotationProcessor "com.uber.nullaway:nullaway:0.4.6"
+  if(JavaVersion.current().getMajorVersion().toInteger() < 9) {
+    // see https://github.com/tbroyer/gradle-errorprone-plugin#jdk-8-support
+    errorproneJavac("com.google.errorprone:javac:9+181-r4173-1")
+  }
+  errorprone("com.google.errorprone:error_prone_core:2.3.2")
+  errorprone("com.uber.nullaway:nullaway:0.4.6")
```

Comment:
Why are you depending on an old version? Also it's not needed in this configuration AFAIK

Reply:
ooops that was a copy paste mistake, thanks for catching it 👍 
one line below is the 0.6.0 ^^